tsconfig.json

```diff
@@ -9,6 +9,9 @@
 		"sourceMap": true,
 		"strictNullChecks": true,
 		"target": "es5",
```

Comment:
Any reason we target such an old version?

Reply:
The initial reason I made this is to have it running in google sheets where the environment is a bit weird (even though it's V8). For example we can't use `const/let` there.

Reply:
I'd appreciate seeing @balena/es-version used but for now this is ok